annotations/annotations_rw_test.go

```diff
@@ -25,7 +26,18 @@ const testRWBody = `[
 	}
 ]`
 
-var expectedRWAnnotations = []*Annotation{
+var expectedReadAnnotations = []*Annotation{
+	{
+		Predicate: "http://www.ft.com/ontology/annotation/mentions",
+		ConceptId: "http://api.ft.com/things/0a619d71-9af5-3755-90dd-f789b686c67a",
```

Comment:
The _expected_ values should certainly be of the form `http://www.ft.com/thing/<uuid>`.